actions/generate-k6-manifests/cmd/generator.go

```diff
@@ -12,6 +13,7 @@ import (
 	"time"
 	"unicode/utf8"
 
+	orderedmap "github.com/elliotchance/orderedmap/v3"
```

Comment:
Curosity:
What's the reason for using oderdedmap instead of just a standard map here?

Reply:
https://github.com/Altinn/altinn-platform/pull/1580#discussion_r2063452980